pkg/action/executor/move_util.go

```diff
@@ -633,57 +638,67 @@ func createClonePod(client *kclient.Clientset, pod *api.Pod,
 		"deployment",        // used by a replicationcontroller to adopt pod
 		"deploymentconfig",  // used by a replicationcontroller to adopt pod
 	}
-	copyPodWithoutSelectedLabels(pod, npod, excludeKeys)
 
-	// copy pod spec, annotations, and labels from the parent
-	kind := parent.GetKind()
-	parentName := parent.GetName()
-	// pod spec
-	podSpecUnstructured, found, err := unstructured.NestedFieldCopy(parent.Object, "spec", "template", "spec")
-	if err != nil || !found {
-		return nil, fmt.Errorf("movePod: error retrieving podSpec from %s %s: %v", kind, parentName, err)
-	}
+	copySpec := false
 	podSpec := api.PodSpec{}
```

Comment:
Nit: this line could also be inside the `else` below right?

Reply:
Updated!